packages/dara-components/changelog.md

```diff
@@ -4,7 +4,9 @@ title: Changelog
 
 ## 1.6.2
 
--   Added `zeroline` default color to `Plotly` theme
+-   Added `zeroline` default color to `Plotly` theme.
+-   Added a new prop `default_legends` to `CausalGraphViewer` and `VisualEdgeEncoder` which allows the user to update the default legends that appear for each `editor_mode`.
+-   Improved typing of `additional_legends`, and added the ability to add nodes as a legend item.
```

Comment:
These should be under `NEXT`

Reply:
Oops, missed the version change in the merge, fixed now